test/EventStore.Client.Tests.Common/EventStoreClientFixtureBase.cs

```diff
@@ -27,11 +30,18 @@ namespace EventStore.Client {
 	public abstract class EventStoreClientFixtureBase : IAsyncLifetime {
 		public const string TestEventType = "-";
 
+		private const string ConnectionString =
+#if GRPC_CORE
+				"esdb://127.0.0.1:2113/?tlsVerifyCert=false"
```

Comment:
@thefringeninja why is this change needed for GRPC_CORE?

Reply:
It was because I did not add the domain name to the security certificate. What's curious is why this worked on `grpc-dotnet`. :shrug: